src/app/about/people.tsx

```diff
@@ -1,76 +1,102 @@
 import Image from "next/image";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 
-const contributors = [
+type Contributor = {
+	name: string;
+	image: string;
+	description: string;
+	email: string;
+};
+
+const contributors = ([] = [
```

Comment:
Creating a specific `Contributor` type is fine if you don't want to inline it but is there any reason you have opted to do `([] = [...])` ?

Reply:
That's probably because you need to to use the type you declared like so: `const contributors: Contributor[] = [...]`.